consul/src/main/resources/config/consul.yml

```diff
@@ -26,6 +26,8 @@ ttlCheck: true
 wait: 600s
 # Additional buffer of time to allow Consul to terminate the blocking query connection
 timeoutBuffer: 5s
+# Response timeout for the non blocking calls
+nonBlockingWait: 5s
```

Comment:
This looks like it is a redundant property. We should consider using requestTimeout in this same consul.yml... It sidesteps all issues of String conversion, and accessor for it already exists in ConsulConfig

Reply:
Code is changed and it uses requestTimeout instead. Part of this PR